src/main/java/org/nest/symboltable/predefined/PredefinedFunctions.java

```diff
@@ -134,13 +134,13 @@
     final MethodSymbol i_sum = createFunctionSymbol(CURR_SUM);
     i_sum.addParameterType(getType("pA"));
     i_sum.addParameterType(getBufferType());
-    i_sum.setReturnType(getType("pA"));
+    i_sum.setReturnType(getType("nS"));
     name2FunctionSymbol.put(CURR_SUM, i_sum);
 
     final MethodSymbol cond_sum = createFunctionSymbol(COND_SUM);
     cond_sum.addParameterType(getType("nS"));
     cond_sum.addParameterType(getBufferType());
-    cond_sum.setReturnType(getType("pA"));
+    cond_sum.setReturnType(getType("nS"));
```

Comment:
in #324 you stated:
cond_sum seems not to deliver nS (I don't know what it does)
neither does curr_sum() seem to deliver pA

from wich i deduced that you expected the curr and cond sum functions to return nS. 
Is the expected behaviour for both of those then to return real?